docs/ldml/tr35-messageFormat.md

```diff
@@ -0,0 +1,69 @@
+## Unicode Technical Standard #35
+
+# Unicode Locale Data Markup Language (LDML)<br/>Part 9: Message Format
+
+|Version|45 (draft)              |
+|-------|------------------------|
+|Editors|Addison Phillips and [other CLDR committee members](tr35.md#Acknowledgments)|
+
+For the full header, summary, and status, see [Part 1: Core](tr35.md).
+
+### _Summary_
+
+This specification defines the data model, syntax, processing, and conformance requirements for the next generation of dynamic messages.
+
+This is a partial document, describing only those parts of the LDML that are relevant for message format. For the other parts of the LDML see the [main LDML document](tr35.md) and the links above.
```

Comment:
This sentence doesn't make sense? There are no parts of LDML here, just the MF2 spec?

```suggestion
For the other parts of LDML see [main LDML document](tr35.md) and the links above.
```

Reply:
Yes, there are other parts of LDML.  This is part 9 of 9 parts.

Reply:
all other parts have the same wording.